src/pybind/rbd/setup.py

```diff
@@ -182,8 +183,32 @@ def cythonize(x, **kwargs):
             return x
 
         source = "rbd.c"
+        Tempita = None
 else:
-    source = "rbd.pyx"
+    # Process Tempita template
+    source_pyx = os.path.join(
+        os.path.dirname(os.path.abspath(__file__)),
+        "rbd.pyx"
+    )
+
+    if Tempita:
```

Comment:
Could you please explain the need for this if statement?  It seems bogus to me because the else clause would be executed only if no exceptions were raised -- I don't see a way for `Tempita` to be None here.